### PR TITLE
feat: Button

### PR DESCRIPTION
**Pull request by Shivanshsharma01**
https://user-images.githubusercontent.com/86787002/154357552-1a777825-be14-4ae9-87e1-2830550faab4.mp4

### CODE REVIEW

File: Components/Button/button.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    padding: .8rem 1rem;
+    border-radius: 1px;
+    border: none;
+    cursor: pointer;
+    transition: all .3s ease-in-out;
+    margin: 10px;
+}
+
+.btn-float {
+
```
Comment (RakshithBhat03) [inline Components/Button/button.css]: Can remove this space. Use a code formatter.

File: Components/Button/button.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
+.btn {
+    font-weight: 600;
+    background: var(--default-color);
+    color: var(--color-white);
+    min-width: 100px;
```
Comment (RakshithBhat03) [inline Components/Button/button.css]: You can use rem unit here.

File: Components/Button/button.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+.btn-link {
+    background-color: var(--primary-color);
+    color: var(--color-white);
+}
+
+.btn-link a {
+    color: var(--color-white);
+}
```
Comment (RakshithBhat03) [inline Components/Button/button.css]: Add new line at EOF

File: Components/Button/button.css [hunk 4]
```diff
@@ -0,0 +1,70 @@
+.btn {
+    font-weight: 600;
+    background: var(--default-color);
+    color: var(--color-white);
+    min-width: 100px;
+    width: max-content;
+    padding: .8rem 1rem;
+    border-radius: 1px;
+    border: none;
```
Comment (nareshpoonia) [inline Components/Button/button.css]: Instead of border: none, you might want to use border transparent
For better accessibility practices

**Review by RakshithBhat03 — COMMENTED** [review]
Suggested some small changes. Apart from that, the code looks well structured and has good use of variables.

**Review by nareshpoonia — APPROVED** [review]
No other comments other than the border, good to merge

**Comment by Shivanshsharma01** [comment]
Added suggested  changes

**Pull request merged**